data/json/professions.json

```diff
@@ -1704,6 +1720,7 @@
     "type": "profession",
     "id": "security",
     "name": "Security Guard",
+    "requirement": "achievement_reach_megastore",
```

Comment:
This profession is the only one allowed for the Trans-Coast Logistics Concourse scenario, which already has the `achievement_reach_trans_coast_lab` requirement, so it could end in the situation where the only profession is still locked behind an achievement even though you already unlocked the scenario...

An option to fix this would be to add the `labtech` and `medic` professions as option to the scenario in question, since those make sense in the context of the scenario and are not locked behind anything yet.

Reply:
Changes Made